docs/maintaining.md

```diff
@@ -63,31 +63,59 @@ docker compose --profile full_stack up -d
 ```
 
 !!! note "Specify the deployment profile"
-    
+
     Whenever you use `docker compose` commands, you also have to specify
     the deployment profile you want to use with the `-p` or `--profile` flag.
     If you forget this step, `docker compose` will use the default profile (`local_node`),
     and will only shut down, update, or start the services for that profile.
 
 ## Updating the data in your graph
 
+If you have followed the [initial setup](getting_started.md)
+and have deployed your Neurobagel node from our Docker Compose recipe,
+your node will have a dedicated graph database that only contains 
+the data for your node.
+
+This makes updating the data in your graph a straightforward process.
+Once you have generated the updated files you want to upload,
+the process to update the data in your graph is:
+
+1. Shut down the Neurobagel node
```

Comment:
same comment as below - the numbered points don't seem to be showing up correctly in the preview 🤔